$(var namespace)/map_merge:
  ros__parameters:
    discovery_rate: 0.5
    map_topic: map
    map_namespace: ssugv
    merged_map_topic: merged_map

    ssugv1/init_pose_x: 0.0
    ssugv1/init_pose_y: 0.0
    ssugv1/init_pose_z: 0.0
    ssugv1/init_pose_yaw: 0.0

    ssugv2/init_pose_x: 0.0
    ssugv2/init_pose_y: 0.0
    ssugv2/init_pose_z: 0.0
    ssugv2/init_pose_yaw: 0.0

    ssugv3/init_pose_x: 0.0
    ssugv3/init_pose_y: 0.0
    ssugv3/init_pose_z: 0.0
    ssugv3/init_pose_yaw: 0.0
